schedule/storage/nfs.yaml

```diff
@@ -15,8 +15,10 @@ conditional_schedule:
         - kernel/nfs_server
       nfs_client:
         - kernel/nfs_client
+    VERSION:
+      15-SP6:
+        - kernel/nfs_stress_ng
```

Comment:
I would do it like https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/5f6c381c38b902af90b1418f4b3f3e3e2b8c9965/schedule/kernel/prepare_baremetal.yaml#L10